README.md

```diff
@@ -29,3 +29,7 @@ Run `make prepare-check` before first `make check`.
 # Contribution notes
 
 - Property should not connect to network.
+
+
+# Contribution guidelines
+*  [CONTRIBUTING.md](/CONTRIBUTING.md)
```

Comment:
Maybe some sentence would match better. What do you think?

Reply:
What is meant by a better matching sentence ? Also what is the /packit copr-build ?

Reply:
The contribution guide is just missing links to core packages that a first timer user would need to intall/go through in order to run. Also some of the instructions seem linux based only.

Reply:
> What is meant by a better matching sentence ?

Use sentence instead of the bullet-point and only the link? (something like `For more info about contributing to our project see our [contribution guide](./CONTRIBUTING.md).`

Reply:
> Also what is the /packit copr-build ?

It is a trigger for our CI. (You are not a member of our GitHub-organisation, and the CI is not triggered automatically.)

Reply:
> The contribution guide is just missing links to core packages that a first timer user would need to intall/go through in order to run. Also some of the instructions seem linux based only.

Could you update that part?